### ISSUE TITLE
Oauth Module

### ISSUE THREAD

**Issue by pablokbg** [freescout-helpdesk/freescout#3750]
Hello,

I reopen issue #3681 , since the solution raised does not work.

If I force the oauth with google, but it fails and I can't login, even if I remove "OAUTHLOGIN_FORCE_OAUTH_LOGIN=1" parameter from .env file and clear cache, I still can't login by the normal way (user and password).

PHP version: 7.3
FreeScout version: 1.6.13
Database: MySQL
Are you using CloudFlare: No

**Comment by twillett** [comment]
I'm having issues with this Oauth module as well. Anyone know what webhooks to use for the various calls? Followed the little instructions via the module info page and it stated the URLs are prefilled in Freescount so I can copy/paste into the Google Dev setup. The fields are not prefilled.

**Comment by freescout-helpdesk** [comment]
Try to clear browser cache.

**Comment by pablokbg** [comment]
I clear cache and switch to another browser and the oauth login persists, is it just removing/commenting the variable? is something else needed?

**Comment by freescout-helpdesk** [comment]
> is it just removing/commenting the variable?

Yes. And clearing the cache: https://github.com/freescout-helpdesk/freescout/wiki/Clearing-the-Cache
